README.md

```diff
@@ -29,8 +30,10 @@ Running Ting
    Django server.
 7. Run the Django server using `python manage.py runserver` inside the `API`
    folder. If it asks you to run migrations, do it.
-8. Install the required dependencies for the node server and the client by running
-   `npm install` in client/ and realtime/. Then `bower install` in client/.
+8. Install node package manager by runnig `sudo apt-get install npm` and bower 
```

Comment:
This guide is meant to be as easy and short as possible. The goal isn't to help you understand, but to help you run ting quickly and painlessly so that you can start developing (and then understand).

This is why, for example, we don't talk about splitting the services into separate servers or whatever – it is the bare minimum.

This is why I think we should merge this.